docs/remix_commands.md

```diff
@@ -3,91 +3,32 @@ Remix Commands
 
 In the console, you can run the commands listed below.  Once you start to type a command, there is *auto completion*.  These commands are using the following libraries:
 
-+ *ethers*: The [ethers.js](https://docs.ethers.io/ethers.js/html/getting-started.html) library is a compact and complete JavaScript library for Ethereum. 
++ *remix*: See the remix commands below that can be run from the console. 
```

Comment:
There should be one line about remix commands and then a link to quickly jump to remix commands section. Same for all.